components/collective-page/hero/Hero.js

```diff
@@ -242,28 +244,22 @@ const Hero = ({ collective, host, isAdmin, onPrimaryColorChange }) => {
                       values={{
                         FiscalHost: <DefinedTerm term={Terms.FISCAL_HOST} color="black.700" />,
                         hostName: (
-                          <LinkCollective collective={host}>
-                            <Span data-cy="fiscalHostName" color="black.700">
-                              {host.name}
-                            </Span>
-                          </LinkCollective>
+                          <OverflowCollectiveName
+                            collective={host}
+                            dataCy="fiscalHostName"
+                            cutOff={parentIsHost && host.name.length >= 30 ? host.name.length : 30}
```

Comment:
We should directly pass the value. The component should be clever enough to truncate only if necessary. I would also use `length` for the prop name, to match lodash's `truncate` function that we're using quite often.

```suggestion
                            cutOff={30}
```